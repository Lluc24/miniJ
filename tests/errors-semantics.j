1 2 3 + 1 2
1 3 { 1 2 3
6 % 0
0 | 5
5 ^ _1
i. _3
