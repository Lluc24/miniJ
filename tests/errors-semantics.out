Error de valor: length error
Error durant l'execuci√≥: index 3 is out of bounds for axis 0 with size 3
0
0
Error de valor: Integers to negative integer powers are not allowed.

